### ISSUE TITLE
Add Support for the Following Languages in the Android Field Officer App

### ISSUE THREAD

**Issue by edcable** [openMF/android-client#703]
A number of languages are most commonly requested by our community. We need to add a new strings.xml for each of these languages so a volunteer can come and translate the strings in that message.

Easy-to-follow instructions on how to Translate Strings provided by @tarun0 at https://github.com/openMF/android-client/wiki/Translation

Support is needed for:

- [ ] Arabic (ar)
- [ ] French (fr)
- [ ] Portuguese (pt)
- [ ] Simplified Chinese (zh-hans)
- [ ] Vietnamese (vi)
- [ ] Nepali (ne)
- [ ] Lao (lo)
- [ ] Indonesia (id)
- [ ] Bengali (bn)
- [ ] Hindi (hi)
- [ ] Tagalog (tl)
- [ ] Kannada (kn)
- [ ] Swahili (sw)
- [ ] Russian (ru)

They should be added under the res directory:

https://github.com/openMF/android-client/tree/master/mifosng-android/src/main/res with the name values-[languagecode].xml

![image 3](https://user-images.githubusercontent.com/823454/27563910-2f590532-5a89-11e7-9fa3-e764ac881964.png)

**Comment by AbhilashG97** [comment]
I'd like to be a volunteer for Hindi (hi) translations for the app. May I proceed ?

**Comment by therajanmaurya** [comment]
@AbhilashG97 go ahead and send PR.

**Comment by tarun0** [comment]
I've added a Wiki page for Translations: https://github.com/openMF/android-client/wiki/Translation with detailed instruction and example. 
@edcable Perhaps you may include it in the issue itself so as to make it convenient for the volunteers to find it.

**Comment by luckyman000786** [comment]
I would like to claim this issue and work on this through GCI. By the Way, I am already a technical volunteer with Mifos (My Name is Chirag Gupta).

**Comment by apoorvinfo** [comment]
Can I work for Hindi and Bengali translation support for the app? @edcable @therajanmaurya @tarun0

**Comment by huangyz0918** [comment]
I’d like to translate Chinese. @edcable @therajanmaurya @tarun0 
This is my first contribution here, should I open an issue for that ?

**Comment by tmudgal** [comment]
@huangyz0918 Yes, please open an issue for Simplified-Chinese and send PR for the same.

**Comment by tarun0** [comment]
@huangyz0918  Yes, please open an issue for Simplified-Chinese and send PR for the same.

**Comment by jasnasuresh** [comment]
I would like to add support for kannada translations in the app. Can I move forward with it?

**Comment by tarun0** [comment]
For Bahasa support, you may also take help of the (closed) PR https://github.com/openMF/android-client/pull/781/

**Comment by PraneethVankayala** [comment]
@tarun0 I will fix that Bahasa issue

**Comment by Arjun2905** [comment]
is this issue still open ?
if yes I would like to work on it .

**Comment by itsPronay** [comment]
still open?
